src/window.rs

```diff
@@ -822,6 +822,16 @@ impl Window {
         self.window.set_minimized(minimized);
     }
 
+    /// Gets the window's current minimized state.
+    ///
```

Comment:
Could you add a note that it shouldn't stop rendering for minimized windows, however you could lower fps for minimize windows.

Not sure how to phrase it better, but you've got an idea.